src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/SearchHelper/WindowsSearchAPI.cs

```diff
@@ -94,7 +88,7 @@ public static void ModifyQueryHelper(ref ISearchQueryHelper queryHelper, string
             }
         }
 
-        public static void InitQueryHelper(out ISearchQueryHelper queryHelper, int maxCount)
+        public void InitQueryHelper(out ISearchQueryHelper queryHelper, int maxCount)

```

Comment:
Can you please pass `DisplayHiddenFiles` as function argument and let this function be static ?

Reply:
Done.